### PR TITLE
Add ScottoAlp Keyboard

### PR DESCRIPTION
**Pull request by joe-scotto**
Add ScottoAlp handwired keyboard

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [ ] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/scottokeebs/scotto34/info.json [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+    },
+    "processor": "RP2040",
+    "url": "https://scottokeebs.com/products/scotto34-pcb-kit",
+    "usb": {
+        "device_version": "1.0.0",
+        "pid": "0x3334",
+        "vid": "0x534B"
+    },
+    "layouts": {
+        "LAYOUT": {
```
Comment (drashna) [inline keyboards/handwired/scottokeebs/scotto34/info.json:38]: This looks like it matches split_3x5_2?

Comment (joe-scotto) [reply keyboards/handwired/scottokeebs/scotto34/info.json:38]: Nope, bottom row has 5 keys... w left, middle space bar, then 2 right of that.

Comment (joe-scotto) [reply keyboards/handwired/scottokeebs/scotto34/info.json:38]: @drashna Nope, bottom row has 5 keys... 2 left, middle space bar, then 2 right of that.

Comment (joe-scotto) [reply keyboards/handwired/scottokeebs/scotto34/info.json]: @drashna Whoops, the Scotto34 directory shouldn't have been in there as it's in other PRs. I removed it in [3918eac](https://github.com/qmk/qmk_firmware/pull/20937/commits/3918eaca1b1ae83d99c27fe28b16800ce4a98a8e).

**Review by dunk2k — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Review by tzarc — APPROVED** [review]

**Pull request merged**